### DISCUSSION TITLE
How to get a PR reviewed on this repo?

### DISCUSSION THREAD

**Discussion by qortex** [nrwl/nx#18362] [General]
I submitted a PR 4 weeks ago, but get no feedback. I understand the volume is high so it's totally ok.
I just wonder if it is expected, or if I shoud register the PR somewhere for someone to be aware it exists?

I think I respected all guidelines [here](https://github.com/nrwl/nx/blob/205b6fec10eb6c592dfe78f5ad75cfd010be1b76/CONTRIBUTING.md)

https://github.com/nrwl/nx/pull/17925

Best,

**Comment by gerarts** [comment]
Have the same question. I found a Slack community with a `#nx-repo` channel, but it's rather quiet there. Posted the same question there, will let you know if I get an update.

For me for #18296 by the way

**Comment by gerarts** [comment]
**Update**

Isaac Mann

>   [3 minutes ago](https://nrwlcommunity.slack.com/archives/CMJ9RH6G6/p1690893811590029?thread_ts=1690875904.211759&cid=CMJ9RH6G6)
> I know it's frustrating to not hear a response.  I don't think you're missing something.
> Feature PRs will naturally get less attention than bug fixes.  I haven't looked at your specific PR, but sometimes people don't respond to feature PRs because they prefer the solution to be done in a different way or they don't think it is worth the cost of maintenance to add the feature, but they don't want to be mean by saying no.
> Also, we work in 2 month cycles.  Our current cycle goes through the end of August and we typically have more time available to go through issues/PRs more thoroughly in the last 2 weeks of the cycle.


Paul Gerarts

>   [2 minutes ago](https://nrwlcommunity.slack.com/archives/CMJ9RH6G6/p1690893835666399?thread_ts=1690875904.211759&cid=CMJ9RH6G6)
> Totally fine :+1: Thanks!
> 
> I think both myself and the poster of the discussion thread are more wondering if we still had to do something (edited) 
> 
> But perfectly happy to just let it sit there until someone picks it up :+1:
> 
> Thanks for the fast reply

**Reply by qortex** [reply]
Thanks!
Yes, I guess a public policy on this would make it clear.
In my case, it's also due to the fact that I need my feature in our workflow. So I wanted to evaluate if I implement it on my side or if the PR was about to go live.

Anyway, thanks for the inquiry!

Apart from my own PR, I feel this policy might drive the number of contributors down: due to this lack of interest, I might not submit another PR on this repo because I don't feel it is welcome. Overall, if all potential contributors feel the same, it drives the number of contributions down. I have little experience on this, maybe it is perfectly fine and the project is better off with internal contributions only. I love Nx either way :)
